google/cloud/spanner_v1/_pandas_helpers.py

```diff
@@ -0,0 +1,76 @@
+import logging
+import six
+
+class pandas_df():
+    
+    def pd_dataframe(self):
+        """Returns the response in a pandas dataframe of the StreamedResultSet object"""
+        try :
+            from pandas import DataFrame
+        except ImportError:
+            logging.error("Pandas module not found. It is needed for converting query result to Dataframe.\n Try running 'pip3 install pandas'")
+
+        code_to_spanner_dtype_dict = {
+                                        1 : 'BOOL',
+                                        2 : 'INT64',
+                                        3 : 'FLOAT64',
+                                        4 : 'TIMESTAMP',
+                                        5 : 'DATE',
+                                        6 : 'STRING',
+                                        7 : 'BYTES',
+                                        8 : 'ARRAY',
+                                        10 : 'NUMERIC'
+                                    }
+        response = six.next(self._response_iterator)
+        if self._metadata is None:  # first response
+            metadata = self._metadata = response.metadata
+            
+        #Creating dictionary to store column name maping of spanner to pandas dataframe
```

Comment:
Nit: spaces after `#` for comments.

Also, this comment is redundant. Comments should show "why" the code is written a certain way, not "what" the code doing.

Reply:
Done